jvm/src/test/scala/com/nvidia/rapids/ml/SparkRapidsMLSuite.scala

```diff
@@ -279,4 +277,59 @@ class SparkRapidsMLSuite extends AnyFunSuite with BeforeAndAfterEach {
     dfCpu.collect()
   }
 
+  test("RapidsLinearRegression") {
+    val df = ss.createDataFrame(
+      Seq(
+        (Vectors.dense(1.0, 2.0), 1.0f),
+        (Vectors.dense(1.0, 3.0), 1.0f),
+        (Vectors.dense(2.0, 1.0), 0.0f),
+        (Vectors.dense(3.0, 1.0), 0.0f))
+    ).toDF("test_feature", "value")
+
+    val rfc = new RapidsLinearRegression()
+      .setFeaturesCol("test_feature")
+      .setLabelCol("value")
+      .setMaxIter(7)
+      .setTol(0.00003)
+
+
+    val path = new File(tempDir, "RapidsLinearRegression").getPath
+    rfc.write.overwrite().save(path)
+
+    val loadedRfc = RapidsLinearRegression.load(path)
```

Comment:
nit:  here too.

Reply:
good catch, I will fix it in the follow up PR. Thx